docker-storage-setup.1

```diff
@@ -63,6 +63,21 @@ GROWPART:
       disabled. Use GROWPART=true to enable automatic partition
       table resizing.
 
+DOCKER_ROOT_VOLUME:
+      If set to "yes", then a separate logical volume will be setup
+      for docker root and it will be mounted at docker root directory.
+      By default, docker root is located at "/var/lib/docker", unless a
+      user start docker daemon using "-g" or "--graph" option and provides
+      a custom docker root location.  
+
```

Comment:
Probably it is a good idea to mention that we are going to scan /etc/sysconfig/docker
file to figure out -g option. If found it will be used as docker root dir otherwise /var/lib/docker is default.